src/power_grid_model/dataset_definitions.py

```diff
@@ -0,0 +1,129 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""Data types for power grid model dataset and component types."""
+
+# This file is automatically generated. DO NOT modify it manually!
+
+import sys
+from typing import Literal, Union
+
+# To avoid conflicts with src/power_grid_model/enum.py
+# pylint: disable=wrong-import-position
+
+sys_path = sys.path.pop(0)
+from enum import Enum
+
+# pylint: enable=wrong-import-position
+
+sys.path.append(sys_path)
+
+# Value names are defined in lower case instead of upper case
+# pylint: disable=invalid-name
+
+
+class DataTypes(Enum):
+    """Dataset types."""
+
+    input = "input"
+
+    sym_output = "sym_output"
+
+    asym_output = "asym_output"
+
+    update = "update"
+
+    sc_output = "sc_output"
```

Comment:
I don't think it is necessary to have empty lines here (and below), there is no logic purpose behind